website/docs/reference/project-configs/target-path.md

```diff
@@ -48,12 +48,22 @@ The precedence order is: CLI flag > env var > `dbt_project.yml`
 </VersionBlock>
 
 ## Examples
-### Use a subdirectory named `compiled` for compiled files
+### Specify subdirectory using the project config file
 
 <File name='dbt_project.yml'>
 
 ```yml
-target-path: "compiled"
+target-path: "compiled_files"
 ```
 
 </File>
+
+<VersionBlock firstVersion="1.5">
+
+### Specify subdirectory from the command line 
+
+```bash
+dbt run --target-path compiled_files
+```
+
```

Comment:
@dbeatty10 : i added an example for `target-path` . look ok? 

maybe it's just me but i couldn't find the page for `profiles-dir` tho 🤷‍♀️

Reply:
Looks good 👍

This is the main thing I found for `--profiles-dir`:
https://docs.getdbt.com/docs/core/connect-data-platform/connection-profiles#advanced-customizing-a-profile-directory